Userland/Applications/PixelPaint/ImageEditor.cpp

```diff
@@ -411,6 +411,9 @@ void ImageEditor::scale_centered_on_position(Gfx::IntPoint const& position, floa
         focus_point.x() - m_scale / old_scale * (focus_point.x() - m_pan_origin.x()),
         focus_point.y() - m_scale / old_scale * (focus_point.y() - m_pan_origin.y()));
 
+    dbgln("scale_centered_on_position(): position={}, size=[{},{}] focus_point={}, scale={}, pan_origin={}",
```

Comment:
I can see these dbgln's become spammy, please gurad them with a define or remove them in the finnished version retroactively

Reply:
Yes, I forgot to remove them :)